### PR TITLE
ptbr translation

### PR DESCRIPTION
**Pull request by mezysinc**
string in ptbr

### CODE REVIEW

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    <string name="enable_socks5_proxy_summary">Quando ativado, todas as conexões de TCP serão redirecionadas ao proxy de SOCKS5 especificado.</string>
+    <string name="proxy">Proxy</string>
+    <string name="proxy_ip_address">Endereço IP de Proxy</string>
+    <string name="proxy_port">Porta de Proxy</string>
+    <string name="root_capture">Capturar via Root</string>
+    <string name="root_capture_summary">A captura de pacotes via Root permite que o PCAPdroid funcione simultaneamente com outros apps de VPN.</string>
+    <string name="donate">Doar</string>
+    <string name="http_request">Solicitação HTTP</string>
+    <string name="root_log">Root Log</string>
+    <string name="request_plaintext">Solicitar texto simples</string>
```
Comment (emanuele-f) [inline app/src/main/res/values-pt-rBR/strings.xml:126]: Here `request` is a noun, not a verb, it's like "the plaintext of the request"

**Review by emanuele-f — CHANGES_REQUESTED** [review]
Thanks for your contribution! Please check my comment below.

File: app/src/main/res/values-pt-rBR/strings.xml (hunk 1 above)
Comment (mezysinc) [reply app/src/main/res/values-pt-rBR/strings.xml:126]: good catch!

Comment (emanuele-f) [reply app/src/main/res/values-pt-rBR/strings.xml:126]: `request_plaintext` was the one to fix, I'll fix it after merge

Comment (emanuele-f) [reply app/src/main/res/values-pt-rBR/strings.xml:126]: See https://github.com/emanuele-f/PCAPdroid/commit/0c3f5a16d55a2fd525c43c3a2f7a16af6102956e

**Pull request merged**